src/Security/Authorization/Core/src/ClaimsAuthorizationRequirement.cs

```diff
@@ -69,5 +70,14 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             }
             return Task.CompletedTask;
         }
+
+        public override string ToString()
+        {
+            var value = (AllowedValues == null || !AllowedValues.Any())
+                ? string.Empty
+                : $" AND Claim.Value in ({string.Join("|", AllowedValues)})";
```

Comment:
nit: " and Claim.Value is one of the following values: ({string.Join("|", AllowedValues)})"